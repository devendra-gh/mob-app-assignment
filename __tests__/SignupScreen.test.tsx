import { render, act } from '@testing-library/react-native';
import SignupScreen from "../screens/SignupScreen";

describe('SignupScreen', () => {
  it('should match snapshot', async () => {
    const result = render(
      <SignupScreen />
    );
    await act(async () => { expect(result).toMatchSnapshot(); })
  });
});

