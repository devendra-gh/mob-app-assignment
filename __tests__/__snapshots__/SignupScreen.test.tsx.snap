// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupScreen should match snapshot 1`] = `
<View
  onLayout={[Function]}
  style={
    [
      {
        "backgroundColor": "#f5f5f5",
        "flex": 1,
        "justifyContent": "center",
        "paddingHorizontal": 20,
      },
      {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <View>
      <View
        accessibilityRole="button"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            [
              {},
            ]
          }
        >
          <Text
            style={
              [
                {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "margin": 8,
                  "textAlign": "center",
                },
              ]
            }
          >
            Send Notification
          </Text>
        </View>
      </View>
    </View>
    <View>
      <Modal
        animationType="slide"
        hardwareAccelerated={false}
        onRequestClose={[Function]}
        transparent={true}
        visible={false}
      />
      <View
        style={
          {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "flex-end",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            [
              {
                "borderRadius": 4,
                "elevation": 2,
                "height": 45,
                "margin": 10,
                "padding": 10,
                "width": 45,
              },
              {
                "backgroundColor": undefined,
              },
            ]
          }
        >
          <Text
            style={
              {
                "color": "#ffffff",
                "fontSize": 16,
                "fontWeight": "bold",
                "textAlign": "center",
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
  <View>
    <View
      style={
        {
          "backgroundColor": "#ffffff",
          "borderRadius": 10,
          "elevation": 5,
          "padding": 20,
          "shadowColor": "#000",
          "shadowOffset": {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.25,
          "shadowRadius": 3.84,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://source.unsplash.com/random?wallpapers",
          }
        }
        style={
          {
            "alignSelf": "center",
            "height": 200,
            "marginBottom": 20,
            "width": "100%",
          }
        }
      />
      <View>
        <Text
          style={
            {
              "fontSize": 14,
              "fontWeight": "bold",
              "marginBottom": 5,
            }
          }
        />
        <TextInput
          autoCapitalize="none"
          onChangeText={[Function]}
          style={
            {
              "borderColor": "#ddd",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 40,
              "marginBottom": 15,
              "padding": 10,
            }
          }
          value=""
        />
      </View>
      <View>
        <Text
          style={
            {
              "fontSize": 14,
              "fontWeight": "bold",
              "marginBottom": 5,
            }
          }
        />
        <TextInput
          autoCapitalize="none"
          onChangeText={[Function]}
          secureTextEntry={true}
          style={
            {
              "borderColor": "#ddd",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 40,
              "marginBottom": 15,
              "padding": 10,
            }
          }
          value=""
        />
      </View>
      <View>
        <Text
          style={
            {
              "fontSize": 14,
              "fontWeight": "bold",
              "marginBottom": 5,
            }
          }
        />
        <TextInput
          autoCapitalize="none"
          onChangeText={[Function]}
          style={
            {
              "borderColor": "#ddd",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 40,
              "marginBottom": 15,
              "padding": 10,
            }
          }
          value=""
        />
      </View>
      <View>
        <Text
          style={
            {
              "fontSize": 14,
              "fontWeight": "bold",
              "marginBottom": 5,
            }
          }
        />
        <TextInput
          autoCapitalize="none"
          onChangeText={[Function]}
          style={
            {
              "borderColor": "#ddd",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 40,
              "marginBottom": 15,
              "padding": 10,
            }
          }
          value=""
        />
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": undefined,
            "borderRadius": 4,
            "marginTop": 10,
            "opacity": 1,
            "padding": 14,
          }
        }
      >
        <Text
          style={
            {
              "color": "#ffffff",
              "fontSize": 16,
              "fontWeight": "700",
              "textAlign": "center",
            }
          }
        />
      </View>
      <View>
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 14,
                  "justifyContent": "center",
                  "paddingTop": 15,
                  "textAlign": "center",
                },
                {
                  "color": undefined,
                },
              ]
            }
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
